### PR TITLE
Terminology updates for images book

### LINKED ISSUE #29595: [enterprise-4.7] Issue in file openshift_images/image-streams-manage.adoc
### Which section(s) is the issue in?

Nav title, assembly title

### What needs fixing?

Is it "imagestreams" or "image streams?"

### PR DESCRIPTION
**Pull request by bmcelvee**
Resolves https://github.com/openshift/openshift-docs/issues/29595. 

Preview links: 
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/configuring-samples-operator.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/samples-operator-alt-registry.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/images-understand.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/create-images.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/managing-images-overview.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/tagging-images.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/image-pull-policy.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/image-streams-manage.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-imagestreams-with-kube-resources.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/triggering-updates-on-imagestream-changes.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/image-configuration.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using-templates.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/templates-using-ruby-on-rails.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/using-images-overview.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/using-images-overview.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins.html
https://deploy-preview-28198--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins-agent.html

### CODE REVIEW

**Comment by bmcelvee** [comment]
Start with using_image -> using-images-overview

**Comment by bmcelvee** [comment]
Resume with image-configuration.adoc

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL, thank you!

**Comment by jboxman** [comment]
@openshift/team-documentation I'll check

**Comment by bmcelvee** [comment]
Thank you, @jboxman! I found and fixed a broken link this morning when adding the preview links, so it is all set. The bulk of the changes is for line break removal + lingering terminology updates (e.g., imagestream -> image stream, ConfigMap -> config map, and so on).

**Comment by jboxman** [comment]
@bmcelvee cool, completed 50% of files so far.

**Comment by bmcelvee** [comment]
Fixed merge conflicts.

**Comment by bmcelvee** [comment]
Hey @jboxman how's the review coming along? I know it's a lot. :sweat_smile:

**Comment by jboxman** [comment]
@bmcelvee 20 files remaining

File: modules/templates-cli-generating-list-of-objects.adoc [hunk 1]
```diff
@@ -57,9 +50,7 @@ $ oc process -f my-rails-postgresql \
     -p POSTGRESQL_DATABASE=mydatabase
 ----
 +
-.. The JSON file can either be redirected to a file or applied directly without
-uploading the template by piping the processed output to the `oc create`
-command:
+.. The JSON file can either be redirected to a file or applied directly without uploading the template by piping the processed output to the `oc create` command:
```
Comment (jboxman) [inline modules/templates-cli-generating-list-of-objects.adoc:53]: This is a holdover from earlier versions, I think, and today it almost certainly outputs YAML.

**Review by jboxman — COMMENTED** [review]
I've left some comments for your review, thanks!

**Pull request merged**